app/views/activity_notification/notifications/default/index.html.erb

```diff
@@ -1,3 +1,6 @@
+
+<% content_for :title, @target.printable_target_name + " - Notifications" %>
```

Comment:
Writing just "Notifications" would be better?

Reply:
I guess no because the other links from the dropdown have the title as `user_name - link`